src/daemon/command_parser_executor.cpp

```diff
@@ -50,7 +50,7 @@ bool t_command_parser_executor::print_peer_list(const std::vector<std::string>&
 {
   if (args.size() > 3)
   {
-    std::cout << "use: print_pl [white] [gray] [<limit>] [pruned] [publicrpc]" << std::endl;
+    std::cout << "Invalid syntax: Too many parameters. For more details, use the help command." << std::endl;
```

Comment:
Actually displaying the output of "help print_pl" here seems better, what do you think? Or at least point to "help print_pl".

Reply:
As a user, I like that better, too. But I also like consistency, and error messages that actually reflect reality. The trouble here is that the output of "help print_pl" is not defined here - the string I'm removing was copy/pasted from `t_command_server` constructor. If print_pl changes in the future, there's some chance that one of these strings would not be updated.

So the right way to get the output here is to get the help strings from `t_command_server` constructor (or the other way around but that would be fighting against epee). Which is tricky, because the command executor gets constructed in the same constructor. Not impossible for sure, but it would require refactoring of the relationship between `t_command_server`, `t_command_parser_executor`, and `epee::console_handlers_binder m_command_lookup`.

Even pointing to "help print_pl" requires copypasta as the command names don't always match the method names in the executor. Pointing to the help command in general was a poor man's attempt in the same direction, without copy pasting strings between files.

So there are three options:

1. Leave it as it is.
2. Copy/paste the command names (low effort).
3. Get the real help strings from `t_command_server` (major refactoring).

Which one would be preferable in your opinion?

Reply:
If it's not easily accessible, then I guess we can kick the can down the road.